src/core/song.cpp

```diff
@@ -1124,6 +1124,20 @@ QString Song::TextForSource(const Source source) {
 
 }
 
+bool Song::SourceIsStreaming(const Source source) {
```

Comment:
Maybe use the existing `Song::is_stream()` function?

Reply:
I think that'll work fine!